src/includes/dinit-iostream.h

```diff
@@ -0,0 +1,130 @@
+#ifndef DINIT_IOSTREAM_H
+#define DINIT_IOSTREAM_H
+
+#include <cstring>
+#include "baseproc-sys.h"
+#include "cpbuffer.h"
+
+// ToDo: Desc
+
+constexpr unsigned BUF_SIZE = 4096; // MAXMUM size of buffers
+
+namespace dio {
+
+template <int fd> class ostream
+{
+    private:
+
+        cpbuffer<BUF_SIZE> buf; // Main buffer
+        bool stdio_sync = false; // Make sure message writed into dest
```

Comment:
syncing with stdio is disabled by default, Is it fine?

Reply:
is this supposed to be the equivalent of `ios_base::sync_with_stdio`? The comment doesn't match that. What do you mean by "sync with stdio"?